contracts/CO30_Matias_Garbageman.Mountains_ACR/edit_me/briefing.sqf

```diff
@@ -0,0 +1,75 @@
+#include "..\do_not_edit\script_component.hpp"
+/*
+ * Author: Kresky, Jonpas
+ * Adds briefing based on side.
+ * Call from initPlayerLocal.sqf.
+ * Copy to other side briefings if you want to add briefings to those sides.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call FUNC(briefing);
+ *
+ * New Line:
+ * <br/>
+ *
+ * Text Colour (Hex colour Code)
+ * <font color='#FF0000'>TEXT HERE</font>
+ *
+ * Clickable Marker Link:
+ * <marker name = 'MARKER_NAME'>MARKER_TEXT</marker>
+ *
+ * Image:
+ * <img image='FOLDER\IMAGE.jpg' width='200' height='100'/>
+ */
+
+params ["_player"];
+
+switch (side _player) do {
+    // BLUFOR briefing
+    case west: {
+        
+    };
+
+    // OPFOR briefing
+    case east: {
+
+    };
+
+    // RESISTANCE/INDEPENDENT briefing -- THIS IS THE BRIEFING YOU WANT TO EDIT FOR YOUR MISSION!
+    case resistance: {
+        _player createDiaryRecord ["Diary", ["Employer", "
+            <br/>NATO
+        "]];
+
+        _player createDiaryRecord ["Diary", ["Situation", "
+            <br/><font color='#D81717'>ENEMY FORCES:</font>
+            <br/>Takistani Militia
+            <br/>AMLCOA: Protect the stolen gear.
+            <br/>
+            <br/><font color='#1D69F6'>FRIENDLY FORCES:</font>
+            <br/>Synixe Contractors
+            <br/>
+            <br/><font color='#993399'>CIVILIAN CONSIDERATIONS:</font>
+            <br/>There is active civilian populace in the AO.
+            <br/>
+            <br/><font color='#663300'>TERRAIN CONSIDERATIONS:</font>
+            <br/>The terrain in the AO is mountainous.
+        "]];
+
+        _player createDiaryRecord ["Diary", ["Mission", "
+            <br/>Two days ago a NATO convoy was ambushed by the Takistani Militia, this ambush resulted in the deaths of ten soldiers and the theft of a bunch of NATO supplies and gear.
```

Comment:
> deaths of ten soldiers

bit extreme, that'd be a very significant altercation

Reply:
Well the NATO soldiers had bad aim